### DISCUSSION TITLE
CLOSE

### DISCUSSION THREAD

**Discussion by rinrinx2** [spencerwooo/onedrive-vercel-index#259] [Q&A]
![Screenshot_3](https://user-images.githubusercontent.com/58888200/148792984-d8fd9568-b194-4ae9-a764-5567ccdd75d0.png)

{"error":{"code":"504","message":"An error occurred with your deployment"}}

**Comment by shaojs321** [comment]
Hi, I met the same problem, how did you solve it

**Reply by rinrinx2** [reply]
https://youtu.be/47cVvI2Kw1M
